classes/components/forms/site/PKPSiteInformationForm.inc.php

```diff
@@ -37,10 +36,12 @@ public function __construct($action, $locales, $site) {
 		$this->action = $action;
 		$this->locales = $locales;
 
-		$this->addField(new FieldTextarea('about', [
+		$this->addField(new FieldRichTextarea('about', [
 				'label' => __('admin.settings.about'),
 				'isMultilingual' => true,
 				'value' => $site->getData('about'),
+				'toolbar' => 'bold italic | link | blockquote bullist numlist',
```

Comment:
I think it's best to leave the toolbar and plugis at their defaults so the field is consistent with the other rich text fields.